gnome-shell/v40/gnome-shell.css

```diff
@@ -2029,21 +2529,9 @@ StScrollBar {
 .openweather-current-icon,
 .openweather-forecast-summary,
 .openweather-forecast-temperature {
-  background: transparent; }
+  background: transparent;
+}
 
 .openweather-current-databox-captions, .openweather-forecast-day {
-  color: rgba(189, 147, 249, 0.5); }
-
-#dashtodockContainer #dash {
```

Comment:
That seems those lines from `extensions/dash-to-dock.scss` were just deleted (not rendered)